.ado/windows-vs-pr.yml

```diff
@@ -465,6 +498,31 @@ jobs:
           version: 0.61.5
           vsComponents: $(VsComponents)
 
+
+  - job: CliInitExperimental
+    displayName: Verify react-native init experimental
+    dependsOn:
+     - Setup
+     - RNWUniversalPR
+    condition: ne( dependencies.Setup.outputs['checkPayload.shouldSkipPRBuild'], 'True' )
+    timeoutInMinutes: 40 # how long to run the job before automatically cancelling
+    cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+    pool:
+      vmImage: $(VmImage)
+    steps:
+      - task: UseNode@1
+        inputs:
+          version: '10.x'
+
+      - template: templates/react-native-init.yml
+        parameters:
+          language: cpp
+          configuration: Release
+          platform: x64
+          version: 0.61.5
```

Comment:
nit: Could we group this with the above version for react-native-init into a variable?